script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -729,12 +729,12 @@ It was fear, together with an inexpressible anger.
 [sha:112a08139e9b7a8da9131eab6317bf071386322c]{
 -- Page 690, Offset 16523.
 -- “だというのに―――<こ|・><の|・><体|・><に|・><は|・>、<あ|・><ま|・><り|・><に|・><も|・><時|・><間|・><が|・><な|・><い|・>”
-'So then why... why does this body of mine have such precious little %{i}time%{/i}?'
+'So then why――why does this body of mine have such precious little %{i}time%{/i}?'
 }
 [sha:ae7fd4443f79965f0e8a4f8d3a506f7e4a8e6015]{
 -- Page 691, Offset 16524.
 -- 世界の広さに比べて<人間|わたし>の生はあまりにも狭すぎる。
-His human lifespan was pitiful in the face of the vastness of the world.
+His human lifespan was pitiful in the face of the world's breadth.
```

Comment:
Kinda prefer vast here

Reply:
"in **the** face of **the** vastness of **the** world" got a bit too much for me.
It's the most common word in english and all, but it got a bit too repetitive here.

Reply:
Yeahhh I can see it. Fair enough.